analyzer/fx/src/main/java/jp/co/ntt/oss/heapstats/plugin/builtin/snapshot/tabs/ReferenceCell.java

```diff
@@ -39,14 +41,25 @@
     private static final long serialVersionUID = -4403355352725440764L;
 
     /**
+     * Defines the font size
+     */
+    private static final String FONT_SIZE =
+        String.valueOf(HeapStatsUtils.getFontSizeOfRefTree());
```

Comment:
Ah, I see. So I will add ReferenceCell#initialize to initialize FONT_SIZE and ZOOM_RATIO by calling RefTreeController#initialize. Any thoughts?